### DISCUSSION TITLE
Deleting resources

### DISCUSSION THREAD

**Discussion by rahul-ve** [Azure/AzOps#581] [Q&A]
Hi

I am trying to understand how to cleanup resources deployed for testing/prototyping. 
We deploy the resources via azops under  the sandbox subscription, what is the process/advise to clean them up and keep azops repo clean.
 
What to do with the resource template files in the repo once the resources were deleted out-of-band. 
Do we delete/`git rm` the ARM templates from under the sandbox subscription scope/folder?

Thanks,
Rahul

**Comment by hazvk** [comment]
Bump on this. 

My experience: I did delete templates and, in conjunction with the setting  `AzOps.Core.CustomTemplateResourceDeletion`, I was able to see resources get torn down. Only minor niggles noted, as per [discussion on sort order with deletes](https://github.com/Azure/AzOps/discussions/878)
